docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.3"
+
+services:
+  REPLACE-SERVICE-NAME:
+    image: maven:3.8.3-openjdk-17
```

Comment:
if we upgrade to jdk 19, change this accordingly

Reply:
confirmed we're sticking to 17